specification/compute/resource-manager/Microsoft.Compute/stable/2022-03-01/ComputeRP/virtualMachineScaleSet.json

```diff
@@ -3507,6 +3507,10 @@
         "disableAutomaticRollback": {
           "type": "boolean",
           "description": "Whether OS image rollback feature should be disabled. Default value is false."
+        },
+        "useRollingUpgradePolicy": {
+          "type": "boolean",
+          "description": "Indicates whether rolling upgrade policy should be used during Auto OS Upgrade. Default value is false."
```

Comment:
Good to add this line too: "AutoOSUpgrade will fallback to the default policy if no policy is defined on the VMSS."

Reply:
Resolved